snooty/rstspec.toml

```
@@ -1709,6 +1709,9 @@ type = {link = "https://learn.mongodb.com%s"}
 help = """Link to the MongoDB Atlas Kubernetes Operator documentation."""
 type = {link = "https://www.mongodb.com/docs/atlas/operator%s", ensure_trailing_slash = true}
 
+[role."languages"]
+type = {link = "https://www.mongodb.com/docs/languages%s", ensure_trailing_slash = true}
```

I believe this should be 
```suggestion
[role."languages"]
type = {link = "https://www.mongodb.com/docs/drivers%s", ensure_trailing_slash = true}
```

The `languages` and `drivers` subdomains are interchangeable for the top level pages only:
i.e. https://www.mongodb.com/docs/languages/kotlin == https://www.mongodb.com/docs/drivers/kotlin

but if you want to reference deeper pages, the `languages` subdomain will 404:
https://www.mongodb.com/docs/languages/kotlin/coroutine/current/quick-start/ vs
https://www.mongodb.com/docs/drivers/kotlin/coroutine/current/quick-start/